### PR TITLE
unset fluid api in paddle.tensor

### PR DESCRIPTION
**Pull request by zhiboniu**
### PR types

Others
### PR changes

Others
### Describe

unset fluid api in paddle.tensor

为了对fluid调用解耦合，逐步减少并消除对fluid的调用
本pr对tensor文件夹下fluid调用进行替换

### CODE REVIEW

File: python/paddle/tensor/search.py [hunk 1]
```diff
@@ -613,7 +614,7 @@ def where(condition, x=None, y=None, name=None):
         broadcast_condition = layers.elementwise_add(cast_cond, broadcast_zeros)
         broadcast_condition = layers.cast(broadcast_condition, 'bool')
```
Comment (jeff41404) [inline python/paddle/tensor/search.py]: from L593 to this line, shall we use paddle.add, paddle.multiply and paddle.cast instead?

**Comment by jeff41404** [comment]
python/paddle/tensor/creation.py L35: from ..fluid.framework import _in_eager_mode
shall we move _in_eager_mode in python/paddle/framework

File: python/paddle/tensor/search.py (hunk 1 above)
Comment (zhiboniu) [reply python/paddle/tensor/search.py]: done

**Comment by zhiboniu** [comment]
> 
yes, we can move all those apis to paddle.framework
from ..fluid.framework import _current_expected_place, _get_paddle_place  # noqa: F401
from ..fluid.framework import dygraph_only  # noqa: F401
from ..fluid.framework import convert_np_dtype_to_dtype_, _varbase_creator, OpProtoHolder  # noqa: F401
from ..fluid.framework import _in_eager_mode  # noqa: F401
from ..fluid.framework import _dygraph_tracer  # noqa: F401

**Review by jeff41404 — DISMISSED** [review]
LGTM

**Review by XiaoguangHu01 — DISMISSED** [review]
LGTM

**Review by jzhang533 — APPROVED** [review]
LGTM

**Review by jzhang533 — APPROVED** [review]
LGTM

**Pull request merged**